web-app/src/components/common/Input/container.js

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+import { PropTypes as PT } from 'prop-types';
+import checkValidity from '../../../utilities/inputValidationRules';
+
+export default Wrapped =>
+  class extends React.Component {
+    static propTypes = {
+      label: PT.string.isRequired,
+      type: PT.string.isRequired,
+      htmlAttrs: PT.object.isRequired,
+      focus: PT.bool,
+      rules: PT.object,
+      validateForm: PT.func,
+      value: PT.any,
```

Comment:
I think there should be defaultProps for any proptype that is not required

Reply:
updated, although couldn't add the console log for the defaultProp function as ESlint doesn't like